### PR TITLE
adding privacy breach section and format fixes

### PR DESCRIPTION
**Pull request by xolotl**

### CODE REVIEW

File: privacy-policy/index.html [hunk 1]
```diff
@@ -61,21 +60,24 @@ <h3 id="accesstoannotations">Access to annotation data</h3>
...
 
-<h3 id="contact">Contact with users</h3>
+<h2 id="contact">Contact with Users</h2>
 
 <p>We send account registration, activation and password reset notifications via email. We also send event notifications such as about replies to annotations via email, with the option to opt-out.  Approximately every quarter we send out a newsletter, “The Margin”, with the option to opt-out.  In the future we may also send information about important service updates or substantial changes in Privacy Policy or our Terms of Service.</p>
 
-<h3 id="consent">Consent</h3>
+<h2 id="breaches">Breaches of Privacy</h2>
+
+<p>In case of a breach of private information, we will notify users within 24 hours.</p>
```
Comment (segdeha) [inline privacy-policy/index.html]: Not sure this wording covers us correctly. 24 hours from when the breach happens? 24 hours from when we become aware of the breach? Notify users how?

Comment (xolotl) [inline privacy-policy/index.html]: Agreed it's minimal. I used this as Lena proposed it and @dwhly +1'd in the original discussion ;)

I'd prefer it to be something more specific too, but I only play a lawyer on TV. Anyone is free to modify this branch to improve the text.

**Comment by dwhly** [comment]
I'm fine w/ a breach clause.  Agree that we should be specific.  24 hours from when we become aware of the breach.  Notify via email.

Sound ok to all?

**Comment by segdeha** [comment]
WFM

**Comment by xolotl** [comment]
Also updated on website at https://hypothes.is/privacy/

**Pull request merged**